src/main/java/org/broadinstitute/hellbender/tools/exome/ReCapSegCaller.java

```diff
@@ -1,92 +1,97 @@
-     * @param zThreshold number of standard deviations from mean required to call an amplification or deletion
      */
-    public static List<ModeledSegment> makeCalls(final TargetCollection<TargetCoverage> targets, final List<ModeledSegment> segments, final double zThreshold) {
+    public static List<ModeledSegment> makeCalls(final TargetCollection<TargetCoverage> targets, final List<ModeledSegment> segments) {
         Utils.nonNull(segments, "Can't make calls on a null list of segments.");
         Utils.nonNull(targets, "Can't make calls on a null list of targets.");
-        ParamUtils.isPositiveOrZero(zThreshold, "zThreshold must be positive or zero.");
-
-        /**
-         * estimate the copy-number neutral log_2 coverage as the median over all targets.
-         * As a precaution we take the median after filtering extreme coverages that are definitely not neutral
-         * i.e. those below -NON_NEUTRAL_COVERAGE or above +NON_NEUTRAL_COVERAGE
-         */
-        final double []  nonExtremeCoverage = targets.targets()
-                .stream()
-                .mapToDouble(TargetCoverage::getCoverage)
-                .filter(c -> Math.abs(c) < NON_NEUTRAL_THRESHOLD)
-                .toArray();
-        final double neutralCoverage = new Median().evaluate(nonExtremeCoverage);
-
-        // Get the standard deviation of targets belonging to high-confidence neutral segments
-        final double[] nearNeutralCoverage = segments.stream()
-                .filter(s -> Math.abs(SegmentUtils.meanTargetCoverage(s, targets) - neutralCoverage) < COPY_NEUTRAL_CUTOFF)
-                .flatMap(s -> targets.targets(s).stream())
-                .mapToDouble(TargetCoverage::getCoverage)
-                .toArray();
-        final double neutralSigma = new StandardDeviation().evaluate(nearNeutralCoverage);
 
+        final double t = calculateT(targets, segments);
+
+        logger.info("Running caller that mimics the ReCapSeg 1.4.5.0 (python) caller.");
+        logger.info(String.format("T in log2CR: %.4f", t));
 
         for (final ModeledSegment segment : segments) {
-            //Get the number of standard deviations the mean falls from the copy neutral value
-            //we should really use the standard deviation of the mean of segment.numTargets() targets
-            //which is sigma/ sqrt(numTargets).  However, the underlying segment means vary due to noise not
-            //removed by tangent normalization, and such a caller would be too strict.
-            //Thus we just use the population standard deviation of targets, and defer a mathematically
-            //principled treatment for a later caller
-            final double Z = (SegmentUtils.meanTargetCoverage(segment, targets) - neutralCoverage)/neutralSigma;
 
-            final String call = Z < -zThreshold ? DELETION_CALL: (Z > zThreshold ? AMPLIFICATION_CALL : NEUTRAL_CALL);
+            String call = NEUTRAL_CALL;
```

Comment:
Haha we have different affection for ternaries.  I don't object to your preference.